Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -166,11 +293,107 @@ def update(self):
 
         self.g.proximal(self.x_tmp, self.tau, out=self.x)
 
-        #update_previous_solution() called after update by base class
-        #i.e current solution is now in x_old, previous solution is now in x
+        # update_previous_solution() #called after update by base class
```

Comment:
can you remove the second `#`, it trips by letting you think it is a comment, while the whole line is a comment